src/buildMathML.js

```diff
@@ -325,6 +325,10 @@ groupTypes.op = function(group) {
         // This is a symbol. Just add the symbol.
         node = new mathMLTree.MathNode(
             "mo", [makeText(group.value.body, group.mode)]);
+    } else if (group.value.value) {
+        // do nothing for now
+        var inner = buildExpression(group.value.value, options);
+        node = new mathMLTree.MathNode("mo", inner);
```

Comment:
I don't know.  To be honest, I wasn't happy with the approach.  Basically, if I remember correctly, I just claimed that some `ParseNode` values may be a list of parse nodes, and if they are then the current parsenode is simply an atom-type wrapper.  But I also haven't thought of any better way.